### PR TITLE
Add runtime theme support for end screen sharing screen

### PR DESCRIPTION
**Pull request by gugalo**
**Jira issue:**
https://glia.atlassian.net/browse/MOB-1969

**Screenshots:**
image
image
image
Screenshot_20230321_113112

### CODE REVIEW

File: widgetssdk/src/main/java/com/glia/widgets/view/unifiedui/exstensions/Merge.kt [hunk 1]
```diff
@@ -30,6 +31,11 @@ internal inline infix fun <reified T : Any?> T.deepMerge(other: T): T {
     return merge(other)
 }
 
+// Added to make method above compatible with Java
+fun deepMerge(old: UiTheme?, other: UiTheme?): UiTheme? {
```
Comment (DavDo) [inline widgetssdk/src/main/java/com/glia/widgets/view/unifiedui/exstensions/Merge.kt:35]: make it internal, please

**Review by DavDo — APPROVED** [review]

File: widgetssdk/src/main/java/com/glia/widgets/view/unifiedui/exstensions/Merge.kt (hunk 1 above)
Comment (gugalo) [reply widgetssdk/src/main/java/com/glia/widgets/view/unifiedui/exstensions/Merge.kt]: Fixed in 56a5746

**Review by Pelkar — APPROVED** [review]
LGTM 👍

**Comment by gugalo** [comment]
!squash

**Comment by gugalo** [comment]
!merge

**Pull request merged**